app/controllers/posts_controller.rb

```diff
@@ -1,9 +1,21 @@
 class PostsController < ApplicationController
```

Comment:
- [ ] New and Create actions are not defined in your _posts controller_. Kindly ensure that the _posts controller_ have _new_ and _create_ actions similar to the one shown below:

```ruby
   def new
      @post = Post.new
    end

    def create
      @post = @current_user.posts.new(post_params)

      respond_to do |format|
        if @post.save
          format.html { redirect_to @post }
        else
          format.html { render :new }
        end
      end
    end

    # remind students about the strong parameters in this lesson https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/lessons/Controllers_lesson.md#:~:text=Pay%20special%20attention%20to%20the%20strong%20parameters%20description%20in%20Rails%20Guides.
    private 
    def post_params
      params.require(:post).permit(:title, :text)
    end

	  ...

  end
```